ansible/roles/vm_set/tasks/main.yml

```diff
@@ -26,11 +65,24 @@
     - libvirt-bin
     - python-libvirt
     - python-pip
+    - apt-transport-https
+    - ca-certificates
+    - curl
+    - software-properties-common
+    - docker-ce
 
 - name: Install python packages
   pip: name=docker-py state=present version=1.7.2
+  become: yes
   environment: "{{ proxy_env | default({}) }}"
 
+- name: Install br_netfilter kernel module
+  become: yes
+  modprobe: name=br_netfilter state=present
```

Comment:
in roles/vm_set/test/start.yml

""
- name: Set sysctl bridge parameters for testbed
  sysctl:
    name: "{{ item }}"
    value: 0
    sysctl_set: yes
  become: yes
  with_items:
   - net.bridge.bridge-nf-call-arptables
   - net.bridge.bridge-nf-call-ip6tables
   - net.bridge.bridge-nf-call-iptables
"""